ui/app/AppLayouts/Wallet/stores/ActivityFiltersStore.qml

```diff
@@ -11,8 +11,8 @@ QtObject {
     property var transactionsList: walletSection.activityController.model
 
     property bool autoUpdateFilter: true
-    property var activityController: walletSection.activityController
-    property bool filtersSet: selectedTime !== Constants.TransactionTimePeriod.All ||
+    readonly property var activityController: walletSection.activityController
```

Comment:
This will make it hard to override in case a stub/mock store is needed, e.g. for Storybook